appeals/web/src/server/appeals/appeal-details/interested-party-comments/add-ip-comment/add-ip-comment.service.js

```diff
@@ -1,31 +1,37 @@
-/** @typedef {import('#appeals/appeal-details/interested-party-comments/interested-party-comments.types').interestedPartyComment} IpComment */
+export const DOCUMENT_STAGE = 'appellant-case';
+export const DOCUMENT_TYPE = 'appellantStatement';
+/** @typedef {import('#appeals/appeal-details/interested-party-comments/interested-party-comments.types').RepresentationRequest} RepresentationRequest */
 
 /**
- * @param { IpComment } comment
  * @param {import('got').Got} apiClient
- * @param {string} document
  * @param {number} appealId
  * @returns {Promise<import('@pins/appeals.api').Appeals.FolderInfo>}
  * */
-export const createIPComment = async (comment, document, apiClient, appealId) => {
-	const json = {
-		ipDetails: {
-			firstName: comment.firstName,
-			lastName: comment.lastName,
-			email: comment.emailAddress
-		},
-		ipAddress: {
-			addressLine1: comment.addressLine1,
-			addressLine2: comment.addressLine2,
-			town: comment.town,
-			county: comment.county,
-			postCode: comment.postCode
-		},
-		attachments: [document],
-		redactionStatus: comment.redactionStatus
-	};
+export const getAttachmentsFolder = async (apiClient, appealId) => {
+	const folders = await apiClient
+		.get(`appeals/${appealId}/document-folders?path=${DOCUMENT_STAGE}/${DOCUMENT_TYPE}`)
+		.json();
+	if (!(folders && folders.length > 0)) {
```

Comment:
Minor: A bit confusing if statement. `if (!folder || !folder.length)` is a bit more straightforward

Reply:
Good catch, this should have been removed as part of a rebase as the `getAttachmentsFolder` function is now in interested-party-comments.service.js